lib/cash-out/cash-out-atomic.js

```diff
@@ -27,6 +27,12 @@ function atomic (tx, pi, fromClient) {
         const isStale = fromClient && oldTx && (oldTx.txVersion >= tx.txVersion)
         if (isStale) throw new E.StaleTxError({ txId: tx.id })
 
+        const isStaleFromServer = !fromClient && oldTx && (oldTx.txVersion > tx.txVersion)
```

Comment:
Why not ``>=``? Also why not throw an error like isStale? Is the default situation that we have an existing oldTx with the identical version?

Reply:
Yeah, I'll add a comment here to explain this. Only the machine bumps tx version, so any server changes will be at best on the same version as the oldTx. 

We don't throw because the job works a bit differently than the machine. The machine needs to know that it's stale, the job should just keep working instead of stopping because of a single bad record.